style.css

```diff
@@ -1,5 +1,235 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap');
+
```

Comment:
- [ ] [OPTIONAL] your website looks fantastic I would love to suggest adding a smoother animation to your website's scroll behavior, try to add `scroll-behavior: smooth;` to your HTML tag in CSS it works like magic! 😮 

``` css
html {
scroll-behavior: smooth;
}
```

Before           | After
:-------------------------:|:-------------------------:
![SUGGGABEFOOOOOOR](https://user-images.githubusercontent.com/89282221/190484838-c64168f1-f1af-4d2a-9c23-8c591cdb2c48.gif) | ![SUGGGAFTERRRRR](https://user-images.githubusercontent.com/89282221/190484864-f35f67af-60d1-4fa8-868d-e34b8d022db2.gif)